seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/config/FlinkJobType.java

```diff
@@ -18,9 +18,7 @@
 package org.apache.seatunnel.core.starter.flink.config;
 
 public enum FlinkJobType {
-    JAR("start-seatunnel-flink.sh"),
-    SQL("start-seatunnel-sql.sh"),
-    ;
+    JAR("start-seatunnel-flink-connector-v2.sh");
 
```

Comment:
Is this still used? flink-starter will only start this shell

Reply:
Yes, I will remove it.